pkg/daemon/update.go

```diff
@@ -271,6 +271,10 @@ func (dn *Daemon) applyOSChanges(oldConfig, newConfig *mcfgv1.MachineConfig) err
 		}
 	}
 
+	if dn.recorder != nil {
+		dn.recorder.Eventf(getNodeRef(dn.node), corev1.EventTypeNormal, "OSUpdateStarted", mcDiff.osChangesString())
```

Comment:
question, do we have an events somewhere that has what we are upgrading _to_?

Reply:
following up on.. myself: in https://github.com/openshift/machine-config-operator/pull/1962/files we have an event that tells us the targeted config but not the OS version - might be nice?

Reply:
We can determine the content by looking at the rendered machineconfigs.  With the events I'm more interested in timing, not content.